.drone.yml

```diff
@@ -100,32 +103,37 @@ build:
       - GOVC_PASSWORD=$GOVC_PASSWORD_ESX
       - if [ "$TAG" != "" ]; then export PKG_VERSION=$TAG; fi
       - #
-      - echo "=> Running tests on ESX_6.5" `date` > /dev/null
+      - echo "=> Build and push vDVS managed plugin" `date` > /dev/null
       - #
       - export GOVC_URL=$GOVC_URL_6_5
       - export VM1=`$GOVC_GET_IP photon.vmfs`
       - export VM2=`$GOVC_GET_IP Ubuntu.16.10`
-      - ./misc/drone-scripts/deploy-and-test-wrapper.sh $GOVC_URL_6_5 $VM1 $VM2 $$BUILD_NUMBER < /dev/null
+      - #
+      - ./misc/drone-scripts/deploy-and-test-wrapper.sh deployplugin $GOVC_URL_6_5 $VM1 $VM2 $$BUILD_NUMBER < /dev/null
+      - #
+      - echo "=> Running tests on ESX_6.5" `date` > /dev/null
```

Comment:
Yes we are. https://github.com/vmware/docker-volume-vsphere/blob/master/misc/drone-scripts/deploy-and-test-wrapper.sh#L54